mite_selenium/__init__.py

```diff
@@ -55,30 +55,43 @@ def stop(self):
 def send_selenium_times(context):
     """Maybe would probably need more error handling
     if somebody tried to use with an incompatible browser"""
-    if context.browser.capabilities['browserName'] != 'chrome':
+    if context.browser.capabilities["browserName"] != "chrome":
         logger.warning(
-            'Timing metrics in %s are not currently supported',
-            context.browser.capabilities['browserName'],
+            "Timing metrics in %s are not currently supported",
+            context.browser.capabilities["browserName"],
         )
         return
 
     timings = context.browser.execute_script(
         'return performance.getEntriesByType("navigation")'
     )[0]
+    request_response_time = timings["responseEnd"] - timings["requestStart"]
+    first_contentful_paint = context.browser.execute_script(
+        'return performance.getEntriesByName("first-contentful-paint")'
+    )[0]["startTime"]
+    time_to_interactive = timings["domComplete"] - first_contentful_paint
```

Comment:
```suggestion
    time_to_interactive = timings["domInteractive"] - first_contentful_paint
```
perhaps a small nit...I'm not totally sure what we are measuring here.  Intuitively it seems like "time that the user can see but not interact with the UI" which seems useful but idk (because I haven't looked) how it fits into accepted metrics of js perf.